webapi/src/main/scala/org/knora/webapi/slice/admin/repo/rdf/Vocabulary.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2021 - 2024 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.slice.admin.repo.rdf
+
+import org.eclipse.rdf4j.model.Namespace
+import org.eclipse.rdf4j.model.impl.SimpleNamespace
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Iri
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf
+
+import org.knora.webapi.messages.OntologyConstants.KnoraAdmin.KnoraAdminPrefixExpansion
+import org.knora.webapi.slice.admin.AdminConstants.adminDataNamedGraph
+
+object Vocabulary {
+  object KnoraAdmin {
+    val NS: Namespace = new SimpleNamespace("knora-admin", KnoraAdminPrefixExpansion)
+
+    // objectClass IRIs
```

Comment:
I would prefer "class IRI" or "resource class IRI", due to the double-meaning of "object" as both "resource" and "last part of a triple". But that's a detail, of course.

Reply:
```suggestion
    // resource class IRIs
```